mllib/src/main/scala/org/apache/spark/ml/r/GBTClassificationWrapper.scala

```diff
@@ -51,6 +51,7 @@ private[r] class GBTClassifierWrapper private (
     pipeline.transform(dataset)
       .drop(PREDICTED_LABEL_INDEX_COL)
       .drop(gbtcModel.getFeaturesCol)
+      .drop(gbtcModel.getLabelCol)
```

Comment:
For all classification algorithms, we do ```forceIndexLabel``` to generate new label column from the original label column, so we need to drop the generated label to make the output more clear to be understand when making prediction.